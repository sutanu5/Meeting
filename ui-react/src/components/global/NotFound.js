import React from 'react';

const NotFound = () => (
  <div id="error-text">
    <div id="error-header">Sorry :(</div>
    Either there was an error in processing or this page does not exist.
  </div>
);

export default NotFound;