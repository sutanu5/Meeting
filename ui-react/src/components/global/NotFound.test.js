import NotFound from './NotFound';

describe('NotFound component', () => {
  let wrapper;
  
  beforeEach(() => {
    wrapper = shallow(<NotFound />);
  });

  it('renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('renders error-text', () => {
    const div = wrapper.find('#error-text').exists();
    expect(div).toBeTruthy();
  });

  it('renders error-text with text', () => {
    const div = wrapper.find('#error-text').text();
    expect(div).not.toBeEmpty();
  })
});