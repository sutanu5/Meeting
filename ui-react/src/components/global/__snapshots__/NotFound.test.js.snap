// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound component renders correctly 1`] = `
<div
  id="error-text"
>
  <div
    id="error-header"
  >
    Sorry :(
  </div>
  Either there was an error in processing or this page does not exist.
</div>
`;
